service/status.go

```diff
@@ -9,16 +9,22 @@ type StatusType uint
 
 // status for services
 const (
-	STOPPED StatusType = 0
-	RUNNING StatusType = 1
-	PARTIAL StatusType = 2
+	STOPPED StatusType = 1
```

Comment:
Why changing this ?
`0` is good like that if it's not initialized then by default the status is `STOPPED`

Reply:
I change it exactly to avoid to have a not initialized version at `STOPPED`.
But now, I'm not sure if it better.

Reply:
I think it's better to revert this because what happen if one moment we don't initialize a status at least it will be marked as stopped and not as nothing that will be really hard to track down